### ISSUE TITLE
Profile not working

### ISSUE THREAD

**Issue by AshHaque** [nuxsmin/sysPass#624]
Hi,

I have installed 2.1.10 yesterday. I created a profile with all the privileges of accounts (no privileges for management, configuration, others). and assign this profile to a user. 

I was expecting this user to be able to edit and delete accounts. but not. Not sure I missed any settings or it's a bug. 

My intention is creating two profile - one with all privileges to manage accounts while other one will have only view privileges. How to achieve that? Can anyone please help me?

Cheers,
Ash

**Comment by nuxsmin** [comment]
Hi, be aware that the user can only view/edit/delete the accounts he (or the group he belongs to) has been granted for. So if you want to grant permissions on all the accounts, you need to switch on the accounts admin option in the user's​ tab.

Please, take a look to the documentation and tell me if there is something missing.

Regards

**Comment by AshHaque** [comment]
I am aware of that. And for this when I login with that user credential I got only few accounts that are associated with the group the user belong to. That's pretty normal. 

I set the profile for this user with view and edit privileges. But I don't get any edit account option for any account when I login with this user credential.

I tried the same in the demo. It's working fine there. But Not sure what the wrong I am doing here.

Cheers,
Ash
